docker-compose.yml

```diff
@@ -0,0 +1,121 @@
+version: '3.6'
+volumes:
+  exec_data:
+services:
+  # database for nisq-analyzer, qc-atlas and pattern-atlas
+  # each container has it's own database, generated by the db image
+  db:
+    build:
+      context: ./db
+    image: quantil-db
+    environment:
+      # specifies user:pw for each container
+      POSTGRES_USERS: 'planqk:planqk|nisq:nisq|patternpedia:patternpedia'
+      # specifies db name for each container
+      POSTGRES_DATABASES: 'planqk:planqk|nisq:nisq|patternpedia:patternpedia'
+      ATLAS_DB: planqk
+      NISQ_DB: nisq
+      PATTERNPEDIA_DB: patternpedia
+    #volumes:
+      # use this volume to save the postgres data
+      # - ./postgres-data:/var/lib/postgresql/data
+    ports:
+      - '5432:5432'
+    networks:
+      - default
+  qc-atlas-ui:
+    image: planqk/qc-atlas-ui
```

Comment:
Add the `latest` image tag here to ensure one always uses the most recent version

Reply:
using the `latest` tag here will not ensure that the latest image version will be used. It will use the locally cached image. To ensure it uses the latest image version, `docker-compose pull` must be used

Reply:
yeah I agree. I'm gonna add the latest tags to avoid any confusion and for consistency